HelpAssistHer/help-assist-her
# Days

## Description
- Addition of days component in pregnancy resource center
- It takes one prop 'hours' from the form.
- It is mapping through it own dayNames array and returns each day name title (left side) with the open and closed status (right side) and timing.
- if open it will show time component with open and close timing otherwise hide it.
- clicking openClose status it will toggle the value 'closedAllDay' parameter in the form field
## Test Plan
- run yarn webpack
- go to http://localhost:4000/verification/pregnancy-resource-center/, login and 'click verify next resource button'. 
- check the Hours input fields
- it will display all days name on left and 'closed' in right will be bright orange if it was closed that day else light orange color.
- click closed it will show and hide open and close time
- update times and open-close status
- save the form
- check on hours array in the network tab
